ui/raidboss/data/triggers/eureka_pyros.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 [{
-  zoneRegex: /(Eureka Pyros|Unknown Zone \(31B\))/,
+  zoneRegex: /(Eureka Pyros|Unknown Zone ID \[31B\])/,
```

Comment:
I've tried going into Eureka Pyros today with the Eureka plugin, but the pyros map didn't load. Then I looked into ACT and it said
Unknown Zone ID [31B] instead of Unknown Zone (31B). Maybe that was changed in the latest version of the ffxiv_ACT_plugin? At least that way it's working for me.

Reply:
Whoa, does Pagos do the same thing?

Reply:
Pagos isn't an unknown zone in ACT anymore, so it is listed as Eureka Pagos in ACT and works without any issues. Only Eureka Pyros does this due to the fact, that it's brand new.